### ISSUE TITLE
SelectCheckboxMenu: 'aria-expanded' value is always set to false

### ISSUE THREAD

**Issue by bobsbrain** [primefaces/primefaces#11018]
### Describe the bug

The 'aria-expanded' attribute for the SelectCheckboxMenu doesn't get updated. No matter if it's opened or not the aria attribute is always set to false.
This problem doesn't occur for other components like the SelectOneMenu.

(Issue can also be seen in the showcase)

### Reproducer

<code><p:selectCheckboxMenu value="#{value}" converter="converter">
&nbsp;&nbsp;&nbsp;&nbsp;<f:selectItems value="#{value}" var="var" itemLabel="#{var.text}" />
&nbsp;&nbsp;&nbsp;&nbsp;<p:ajax event="change" listener="#{handler.onSelect}" update="@(#id)" partialSubmit="true"/>
</p:selectCheckboxMenu></code>

### Expected behavior

The aria attribute should get updated accordingly to the state of the component. True when it's opened, false when it's closed.

### PrimeFaces edition

Community

### PrimeFaces version

13.0.3

### Theme

Saga

### JSF implementation

Mojarra

### JSF version

2.3.18

### Java version

17

**Comment by melloware** [comment]
The aria-expanded actually gets toggled on the hidden input which controls the checkbox

![image](https://github.com/primefaces/primefaces/assets/4399574/83140d24-1efc-4a15-bbb9-5ef3038940d8)

I have to look it might accidentially be on the outer Div when its controlled at the Input level?

**Comment by melloware** [comment]
It looks like this SelectCheckBoxMenu is a little twisted around as teh hidden KeyBoard trap handles opening and closing the popup with `aria-controls` and `aria-expanded` but its marked `aria-hidden` so a screenreader will ignore this.  @Bukama @christophs78 we may need some guidance on what is the proper ARIA for this control?

**Comment by bobsbrain** [comment]
Yeah, for the SelectOneMenu im selecting the aria-expanded attribute in the outer div, as the hidden select element doesn't have the aria-expanded attribute. For the SelectCheckboxMenu it seems that both the outer div, as well as the hidden input element have an aria-expanded attribute but only the latter gets updated accordingly.
Updating the attribute on the outer div seems more practical to me.

**Comment by melloware** [comment]
Agreed let me look through commit history and figure out why I did that.  Will fix it for 13.0.4
